src/Fabulous.XamarinForms/Views/_View.fs

```diff
@@ -62,6 +62,26 @@ module XFView =
                     view.TranslateTo(data.X, data.Y, data.AnimationDuration, data.Easing)
                     |> ignore)
 
+    let TranslateXTo =
+        Attributes.defineSimpleScalarWithEquality<float>
+            "View_TranslateXTo"
+            (fun _ newValueOpt node ->
+                let view = node.Target :?> View
+
+                match newValueOpt with
+                | ValueNone -> view.TranslationX <- 0.
```

Comment:
Unlike ScaleX/ScaleY/RotateX/RotateY, TranslationX/TranslationY don't have corresponding methods with animation.
And since they are BindableProperties, we need to call `view.SetValue(VisualElement.TranslationXProperty, value)` / `view.ClearValue(VisualElement.TranslationXProperty)` instead of direct assignments

Reply:
I guess you can replace everything with `Attributes.defineBindableFloat`

Reply:
Done :)